seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -1867,6 +1867,7 @@
       ],
       "tests": [
         "assert((function(d){if(d[0][0] == 'John' && d[0][1] == 23 && d[2] == undefined){return true;}else{return false;}})(myArray), 'message: <code>myArray</code> should only contain <code>[[\"John\", 23]]</code>.');",
```

Comment:
First test gives false positive under the conditions or the original issue. Can you please change `if(d[0][0] == 'John' && d[0][1] == 23 && d[2] == undefined` to `if(d[0][0] == 'John' && d[0][1] == 23 && d[1] == undefined` so that it checks the correct part of `myArray`